index.js

```diff
@@ -20,3 +20,30 @@ module.exports = fn => {
 
 	return ret;
 };
+
+const log = (fn, promise) => {
+	console.log(`Promise from ${fn.displayName} resolved in ${promise.time} ms`);
```

Comment:
`fn.displayName` => `fn.name` and fallback to `[anonymous]` when it's an empty string.